### ISSUE TITLE
BUG: error when scan/rescan reaches API quote

### ISSUE THREAD

**Issue by dav10re** [pwelch/virustotal_api#20]
Hello there, I've found a bug when I tried to rescan an URL with 

* VirusTotalAPI::URLScan.scan 
* VirusTotalAPI::FileRescan.rescan

If you try to do more requests togheter until you reach API quota limits, you will have a "JSONParser Error: unsepected token at ' ' ". This happens because when you reach API limit the response will be without content (in fact you will obtain a 204 http code and your body will be empty).

If you want to  analyze the error you can use VirusTotal API:

```ruby
require 'rest-client'

url = 'https://www.virustotal.com/vtapi/v2/url/report'
response = RestClient.post(url, {'apikey': YOUR_API, 'resource': URL_TO_SUBMIT})
puts response.body
puts response.code
puts response.headers
```
Thanks.

**Comment by jonnynux** [comment]
I'll fix it.

**Comment by pwelch** [comment]
@jonnynux's patch has been merged and a new version released (v0.4.1) 

https://rubygems.org/gems/virustotal_api

Thanks for reporting the issue!
